undefined: syscall.Kill when compiling on windows v1.4

### Expected behavior
No errors compiling.

### Observed behavior
When compiling on windows 

> vendor\github.com\bugsnag\bugsnag-go\sessions\tracker.go:133:2: undefined: syscall.Kill

### Steps to reproduce
Compile on windows 

### Version
V 1.4.0

### Additional information
From some quick googling looks like syscall.Kill can be replaced with the following to work on windows:
```
p, err := os.FindProcess(os.Getpid())

if err != nil {
	return err
}

return p.Signal(syscall.SIGTERM)
```

--- a/sessions/tracker.go
+++ b/sessions/tracker.go
@@ -130,7 +130,12 @@ func (s *sessionTracker) flushSessionsAndRepeatSignal(shutdown chan<- os.Signal,
 			s.config.logf("%v", err)
 		}
 	}
-	syscall.Kill(syscall.Getpid(), sig)
+
+	if p, err := os.FindProcess(os.Getpid()); err != nil {
+		s.config.logf("%v", err)
+	} else {
+		p.Signal(sig)
+	}
 }
 
 func (s *sessionTracker) FlushSessions() {

